src/graph_notebook/notebooks/04-Machine-Learning/Neptune-ML-06-Text-Encoding-Tutorial.ipynb

```diff
@@ -0,0 +1,1281 @@
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "1093ff9b0ecd44e0959f3a7f0911ed42",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Tab(children=(Output(layout=Layout(max_height='600px', overflow='scroll', width='100%')), Output(layout=Layout…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%gremlin\n",
+    "g.E('app_84898').drop()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "5db08fcfe45f4b81aff9dd520f543114",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Tab(children=(Output(layout=Layout(max_height='600px', overflow='scroll', width='100%')), Output(layout=Layout…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%gremlin\n",
+    "g.V('job_26829').valueMap('Title')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "55b3bae1dc354cac8d3470ca7f33447e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Tab(children=(Output(layout=Layout(max_height='600px', overflow='scroll', width='100%')), Output(layout=Layout…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%gremlin\n",
+    "g.E('app_5325').drop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Checking our data again, we see that two of five edges have now been removed. Only three edges are left."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "a43d985a5e3846f2a94c0e61aa5385d8",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Tab(children=(Output(layout=Layout(max_height='600px', overflow='scroll', width='100%')), Output(layout=Layout…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%gremlin\n",
+    "g.V('user_443931').out('apply').hasLabel('job').valueMap('Title')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 2: Export Data: Choose the Right Text Encoders for Text Properties\n",
+    "\n",
+    "**Note:** Before exporting data ensure that Neptune Export has been configured as described here: [Neptune Export Service](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-data-export-service.html#machine-learning-data-export-service-run-export). \n",
```

Comment:
Does the link still exist? It redirects to "What is Amazon Neptune?"

Reply:
Updated the link to be https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-data-export.html

Reply:
The process is automatic. We've provided the configuration.

Reply:
The current note is a bit misleading: "Note: Before exporting data ensure that Neptune Export has been configured as described here: [Neptune Export Service](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-data-export.html)."

 If we've already configuring this for the customer, we should say so. e.g., "For this tutorial, we have pre-configured ....... When using Neptune ML on the export step for your use case, ensure that Neptune Export has been configured as described here."

Reply:
I think I misunderstood the previous comment 'Is it possible to automate this using CFN within the notebook so that customers have to do least amount of configuration/setup as possible? Or is that already provided in the below cells?'. The customer needs to configure the Neptune Export Service based on the manual here https://docs.aws.amazon.com/neptune/latest/userguide/export-service.html. Also, this is what other notebook describes, like in https://github.com/aws/graph-notebook/blob/main/src/graph_notebook/notebooks/04-Machine-Learning/Neptune-ML-01-Introduction-to-Node-Classification-Gremlin.ipynb 

Btw, the sentence here is correct. It is a note and also provides the link to help check the configuration of the Neptune Export Service.